src/Raven.Client/Extensions/ChangeVectorExtensions.cs

```diff
@@ -105,5 +105,42 @@ public static ChangeVectorEntry[] ToVector(this BlittableJsonReaderArray vectorJ
             }
             return result;
         }
+
+        public static string ToJson(this ChangeVectorEntry[] self)
+        {
+            if (self == null)
+                return null;
+
+            var sb = new StringBuilder();
+            for (int i = 0; i < self.Length; i++)
+            {
+                if (i != 0)
+                    sb.Append(", ");
+                self[i].Append(sb);
+            }
+            return sb.ToString();
+        }
+
+        public static void ToBase26(StringBuilder sb, int tag)
+        {
+            do
+            {
+                var reminder = tag % 26;
+                sb.Append((char)('A' + reminder));
+                tag /= 26;
```

Comment:
Switch these two lines, it will let the JIT produce nicer code (just one `idiv` instruction)